File: Suhyun/Level1/12월 5주차/문자열 다루기 기본/String_Control_Default.java
public class String_Control_Default {
    public static void main(String[] args) {
        String s = "1234231";
        boolean answer =true;

        if( (4 == s.length() || s.length() == 6) && s.matches("^[0-9]+$") == true)
        {
            System.out.println(true);
            //return answer;
        }
        else {
            System.out.println(false);
            //return false;
        }
        
    }
    
}

File: Suhyun/Level1/12월 5주차/문자열 다루기 기본/String_Control_Default.js
function solution(s) {
    let answer = true;
    let reg = /^[0-9]*$/;
         
    if(reg.test(s) == true && (s.length == 4 || s.length == 6))
    {
         return true;
    }
    else{
      return false;
    }
}